packages/grafana-ui/src/components/Forms/Select/SelectBase.tsx

```diff
@@ -178,6 +178,15 @@ export function SelectBase<T>({
 }: SelectBaseProps<T>) {
   const theme = useTheme();
   const styles = getSelectStyles(theme);
+  const onChangeWithEmpty = useCallback(
+    (value: SelectValue<T>) => {
+      if (isMulti && !Array.isArray(value)) {
+        return onChange([]);
```

Comment:
Shouldn't the value be in the call?
```suggestion
        return onChange([value]);
```

Reply:
ah, nope - because it is undefined or null in that scenario.

Reply:
bit I guess I need to make that a bit clearer in the code.

Reply:
maybe check for undefined instead?

Reply:
yeah, I will update this.

Reply:
good feedback :)